src/apps/testapps/testH3Api.c

```diff
@@ -47,4 +49,34 @@ TEST(geoToH3_coord) {
              "coordinates with infinity are rejected");
 }
 
+TEST(h3ToGeoBoundary_classIIEdgeVertex) {
```

Comment:
should this be `classIIIEdge` (not `classIIEdge`)?  unless i misunderstood issue...